aip/0135.md

```diff
@@ -179,6 +179,39 @@ message DeleteBookRequest {
 If the etag is provided and does not match the server-computed etag, the
 request **must** fail with a `FAILED_PRECONDITION` error code.
 
+### Deletion prevention
+
+Sometimes, it may be desirable for users to protect a resource against accidental
+deletion. This is often especially useful in "infrastructure as code" deployments
+where resources are kept in sync with some declarative configuration.
+
+To achieve this, the resource **should** include either a `bool prevent_delete`
```

Comment:
We should pick one and only one. (I think `prevent_delete` is our only option given that we have true as abnormal-polarity everywhere and `allow_delete` would be a breaking change.)

Reply:
Fair enough. That certainly simplifies things.